crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/ContributionService.java

```diff
@@ -176,4 +176,17 @@ public boolean isCds15WorkAround(final RepOrderDTO repOrderDTO) {
         return PassportAssessmentResult.FAIL.getResult().equals(passportAssessmentResult)
                 && InitAssessmentResult.PASS.getResult().equals(initialAssessmentResult);
     }
+
+    public boolean hasMessageOutcomeChanged(int repId, String laaTransactionId, String msgOutcome) {
```

Comment:
I believe at this point we would have the RepOrderDTO from the checkReassessment method. 
We can update the method signature to use RepOrderDTO as the input.

Reply:
This will be called from compare_contribution, which does not have RepOrderDTO.